### PR TITLE
Fix notebook controller chart

### PR DESCRIPTION
**Pull request by akashg3627**

### CODE REVIEW

File: charts/tfy-notebook-controller/templates/notebook-controller-standalone.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ metadata:
     app: notebook-controller
     control-plane: controller-manager
     kustomize.component: notebook-controller
-  name: nb-ctl-system
+  name: {{ .Release.Namespace }}
```
Comment (dunefro) [inline charts/tfy-notebook-controller/templates/notebook-controller-standalone.yaml:8]: We should remove the creation of namespace from the chart

**Review by dunefro — CHANGES_REQUESTED** [review]

File: charts/tfy-notebook-controller/templates/notebook-controller-standalone.yaml [hunk 2]
```diff
@@ -5,7 +5,7 @@ metadata:
     app: notebook-controller
```
Comment (debajyoti-truefoundry) [inline charts/tfy-notebook-controller/templates/notebook-controller-standalone.yaml:5]: Do we need this whole block?

Comment (debajyoti-truefoundry) [reply charts/tfy-notebook-controller/templates/notebook-controller-standalone.yaml:5]: In the flow, namespace will be already created, right?

Comment (debajyoti-truefoundry) [reply charts/tfy-notebook-controller/templates/notebook-controller-standalone.yaml:5]: Is this required to add the labels?

**Pull request closed without merging**